t5/t5.py

```diff
@@ -287,6 +289,10 @@ def decode(
         y, cache = self.decoder(
             inputs, memory=memory, mask=mask, memory_mask=None, cache=cache
         )
+        if self.tie_word_embeddings:
+            # Rescale output before projecting on vocab
+            # See https://github.com/huggingface/transformers/blob/71d47f0ad498b7649f11d3a9cca3cd3585e4341f/src/transformers/models/t5/modeling_t5.py#L1766C9-L1769C71
+            y *= self.model_dim ** -0.5
```

Comment:
Good find! Does this account for the whole discrency?

Reply:
It does not, but I finally managed to fix it in https://github.com/ml-explore/mlx-examples/pull/113/commits/64e53e84158e5c2e712de10566395c5b8a3edf5f

Reply:
Oh, btw, we have the same bug in mlx: https://github.com/ml-explore/mlx/pull/210